DDRec/include/DDRec/DCH_info.h

```diff
@@ -0,0 +1,348 @@
+#ifndef DCH_INFO_H_INCLUDED
+#define DCH_INFO_H_INCLUDED
```

Comment:
```suggestion
#ifndef DDREC_DCH_INFO_H
#define DDREC_DCH_INFO_H
```

The clang-tidy checker doesn't fail :/